### ISSUE TITLE
PHPUnit stopped working with Luthier-CI

### ISSUE THREAD

**Issue by Ihabafia** [ingeniasoftware/luthier-ci#31]
Dear Sir,

Once I installed the Luthier-CI, all the tests of the TDD UnitTest start giving the following error:

```
1) Welcome_test::test_index
Exception: Serialization of 'Closure' is not allowed
```

One I comment the Hook information:

```
$hook = Luthier\Hook::getHooks([
    'modules' => ['auth']
]);
```

Everything works fine, any suggestion?

Thank you,

**Comment by Ihabafia** [comment]
Any News???

**Comment by andersonsalas** [comment]
Hi!

The unit tests are not being finished yet. we were very busy but the development of this project (and others that we own) will be resumed in a few days. However, these unit tests are meant for the Luthier CI development, not to be used as a PHPUnit bridge for CodeIgniter applications. You can use the [kenjis/ci-phpunit-test](https://github.com/kenjis/ci-phpunit-test) package for that.

**Comment by Ihabafia** [comment]
I did, but all the tests fails because of the error that I told you about:

```
1) Welcome_test::test_index
Exception: Serialization of 'Closure' is not allowed
```

**Comment by andersonsalas** [comment]
Yeah, I've confirmed the error, but it's too early to tell you what's triggering it. At the moment I'll keep this issue open while finding a solution :worried:

**Comment by andersonsalas** [comment]
Nothing yet. I'm still looking for a solution.

**Comment by yahyaerturan** [comment]
Is this repo is still being maintained?

**Comment by andersonsalas** [comment]
> Is this repo is still being maintained?  

Hi @yahyaerturan!. Yes, the repo is still maintained, however, no more features are planned to add for now.

**Comment by banqhsia** [comment]
Related to https://github.com/kenjis/ci-phpunit-test/issues/187#event-1118117087

It can be solved by the workaround above.
